diff --git a/concolor.d.ts b/concolor.d.ts
index 1741e1c3f5e3794592d60b5b34a69537da30d0f4..f51fc48413ff612acd473597aa8af7a0eb084c16 100644
--- a/concolor.d.ts
+++ b/concolor.d.ts
@@ -1,24 +1,30 @@
-type Strings = string | Array<string>;
-type TagFuction = (strings: Strings, ...values: Array<any>) => string;
+type ConcolorFunction = {
+  (strings: TemplateStringsArray, ...subs: any[]): string;
+  (string: string): string;
+}
 
 interface Concolor {
-  b: TagFuction;
-  i: TagFuction;
-  u: TagFuction;
-  em: TagFuction;
-  error: TagFuction;
-  info: TagFuction;
-  warn: TagFuction;
-  debug: TagFuction;
-  succes: TagFuction;
-  fail: TagFuction;
-  red: TagFuction;
-  green: TagFuction;
-  yellow: TagFuction;
-  blue: TagFuction;
-  magenta: TagFuction;
-  cyan: TagFuction;
-  white: TagFuction;
+  b: ConcolorFunction;
+  i: ConcolorFunction;
+  u: ConcolorFunction;
+  em: ConcolorFunction;
+
+  error: ConcolorFunction;
+  info: ConcolorFunction;
+  fail: ConcolorFunction;
+  warn: ConcolorFunction;
+  debug: ConcolorFunction;
+  success: ConcolorFunction;
+  
+  red: ConcolorFunction;
+  green: ConcolorFunction;
+  yellow: ConcolorFunction;
+  blue: ConcolorFunction;
+  magenta: ConcolorFunction;
+  cyan: ConcolorFunction;
+  white: ConcolorFunction;
 }
 
-export const concolor: TagFuction | Concolor;
+declare const concolor: ConcolorFunction & Concolor;
+
+export default concolor;
\ No newline at end of file